frontend/components/panels/WalletDaiDepositCheckPanel.stories.js

```diff
@@ -0,0 +1,97 @@
+import { storiesOf } from '@storybook/vue';
+import { action } from '@storybook/addon-actions';
+import faker from 'faker';
+import BigNumber from 'bignumber.js';
+import WalletDaiDepositCheckPanel from '~/components/panels/WalletDaiDepositCheckPanel';
+
+const common = {
+    components: { WalletDaiDepositCheckPanel },
+    data() {
+        return {
+            walletDai: new BigNumber(faker.finance.amount(0, 100)),
+            desiredAmount: new BigNumber(faker.finance.amount(101, 200)),
+            network: 'main',
+            tokenAddressDai: faker.finance.ethereumAddress(),
+            isExplanationsShown: true,
+            disabled: false,
+            isLoading: false,
+        };
+    },
+    methods: {
+        refresh() {
+            this.isLoading = true;
+            setTimeout(() => {
+                this.isLoading = false;
+            }, 1000);
+        },
+        isCorrect: action('isCorrect'),
+    },
+    template: `
+        <WalletDaiDepositCheckPanel
+            v-bind="$data"
+            @refresh="refresh"
+            @update:isCorrect="isCorrect"
+        />`,
+};
+
+storiesOf('Panels/WalletDaiDepositCheckPanel', module)
+    .add('Insufficient amount of DAI present', () => ({
+        ...common,
+    }))
+    .add('Sufficient amount of DAI present', () => ({
+        ...common,
+        data: () => ({
+            ...common.data(),
+            walletDai: new BigNumber(100),
+            desiredAmount: new BigNumber(10),
+        }),
+    }))
+    .add('Missing Desired Amount', () => ({
+        ...common,
+        data: () => ({
+            ...common.data(),
+            desiredAmount: undefined,
+        }),
+    }))
+    .add('Desired Amount is NaN', () => ({
+        ...common,
+        data: () => ({
+            ...common.data(),
+            desiredAmount: new BigNumber(NaN),
+        }),
+    }))
+    .add('Desired Amount is negative', () => ({
+        ...common,
+        data: () => ({
+            ...common.data(),
+            desiredAmount: new BigNumber(-10),
+        }),
```

Comment:
I wonder if this is actually properly handled by the panel 😅

Reply:
It is! That is also why I added the story 😄 

See: https://github.com/sidestream-tech/unified-auctions-ui/blob/main/frontend/components/panels/WalletDaiDepositCheckPanel.vue#L87

Reply:
I remember I did it somewhere, but I don't remember where exactly 🔍

Reply:
If I ever have some free time, I might go through and see if we are missing stories for any other components. 

I agree that a higher coverage rate would be nice to have.